create-neon/src/shell.ts

```diff
@@ -1,11 +1,11 @@
-import { spawn } from 'child_process';
-
+import { spawn } from "child_process";
+import fs from "fs/promises";
```

Comment:
Unfortunately, the promises API is only exposed at `fs/promises` in newer node versions. In order to support older versions, this will need to be updated:

```suggestion
import { promises as fs } from "fs";
```

This can be seen in the failed CI unit tests.